upstream-community-operators/postgresql-operator/0.0.1/postgresql-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,177 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "postgresql.example.com/v1alpha1",
+          "kind": "Postgresql",
+          "metadata": {
+            "name": "example-postgresql"
+          },
+          "spec": {
+            "image": "postgres:10.4",
+            "size": 3
+          }
+        }
+      ]
+    capabilities: Basic Install
+  name: postgresql-operator.v0.0.1
```

Comment:
There is an Operator with this name already here https://github.com/operator-framework/community-operators/blob/master/upstream-community-operators/postgresql-operator-dev4devs-com/postgresql-operator-dev4devs-com.package.yaml